### PR TITLE
feat(storybook): add a extension-tables story

### PR DESCRIPTION
**Pull request by ocavue**
### Description

Add a `TablesExtension` story into the storybook.

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [x] I have updated the documentation where necessary.
- [ ] New code is unit tested and all current tests pass when running `pnpm test`.

### Screenshots

https://user-images.githubusercontent.com/24715727/109177203-82fe5980-77c2-11eb-99f1-280ac07cd5eb.mov

### CODE REVIEW

File: examples/storybook-react/src/table.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          add a row before the current one
+        </button>
+
+        <button onClick={() => commands.deleteTable()}>delete the table</button>
+      </p>
+    </div>
+  );
+};
+
+const InnerEditor = () => {
```
Comment (ocavue) [inline examples/storybook-react/src/table.stories.tsx:70]: I don't know if this is the idea way to build a editor.

**Review by ifiokjr — APPROVED** [review]
Beautiful 😍

**Comment by ifiokjr** [comment]
Are you happy to merge this?

**Comment by ocavue** [comment]
> Are you happy to merge this?

@ifiokjr I have no further changes planned for this PR. If your think the code is ok, then it's ready for merge. (please remember to use Squash Merge to override my ugly Git commit history 😅)

**Pull request merged**